app/javascript/components/visualization/ScatterPlot.js

```diff
@@ -298,6 +300,11 @@ function traceNameSort(a, b) {
   return a.localeCompare(b)
 }
 
+/** sort alphanumerically */
+function sortAlphaNumerically(a, b) {
```

Comment:
"[Natural sort](https://en.wikipedia.org/wiki/Natural_sort_order)" is the conventional term for this kind of sort, so preferring that name would help maintainability.

```suggestion
/** Give natural sort order for alphanumeric strings */
function sortNaturally(a, b) {
```